bambi/tests/test_built_models.py

```diff
@@ -71,6 +71,15 @@ def inhaler():
     return data
 
 
+@pytest.fixture(scope="module")
+def categorical_family_categorical_predictor():
+    from os.path import dirname, join
```

Comment:
Why import these here and not in global namespace?

Reply:
There's no real reason. They just have been there forever. But this is a good time to move them to the global namespace.